plugins/css-blank-pseudo/src/index.ts

```diff
@@ -2,7 +2,7 @@ import type { PluginCreator } from 'postcss';
 import parser from 'postcss-selector-parser';
 import isValidReplacement from './is-valid-replacement.mjs';
 
-type pluginOptions = { color?: string, preserve?: boolean };
+type pluginOptions = { preserve?: boolean, replaceWith?: string };
```

Comment:
I realised this was a leftover which would have exported the wrong types, this hasn't been published yet so it's not a big concern